nltk/tbl/__init__.py

```diff
@@ -14,16 +14,15 @@
 currently used by nltk.tag.BrillTagger.
 """
 
+from nltk.tbl.erroranalysis import error_list
```

Comment:
Perhaps this file would rather not be sorted? Considering it had comments for each import previously.
That said, the comments aren't particularly useful or anything.

Reply:
good catch! haha may as well skip it